src/Components /ArticleVotes.jsx

```diff
@@ -0,0 +1,70 @@
+import { useState, useEffect } from 'react';
+import * as api from '../api.js';
+
+
+export default function ArticleRating ({ articleId, articleVotes }){
+  const [votes, setVotes] = useState(articleVotes);
+  const [incVotes, setIncVotes] = useState(0);
+  const [voteIncrease, setVoteIncrease] = useState(false);
+  const [voteDecrease, setVoteDecrease] = useState(false);
```

Comment:
I think both of these states actually could be better named 'disableVoteIncrease' etc. 
Because they're completely used to determine if it should be disabled or not and reads clearer